test/v3/routes/tracks_test.rb

```diff
@@ -27,4 +27,28 @@ def test_tracks
     blueberry = JSON.parse(last_response.body)["track"]
     Approvals.verify(blueberry, name: 'v3_track_unimplemented')
   end
+
+  def test_readme
+    get '/tracks/fake/three/readme'
+
+    options = { format: :json, name: 'v3_valid_readme' }
+    Approvals.verify(last_response.body, options)
+  end
```

Comment:
If you have a couple of hours to spare, there's a video of a fantastic workshop from LinuxConf Australia called Git for Ages 4 and Up: https://www.youtube.com/watch?v=1ffBJ4sVUb4